### DISCUSSION TITLE
ggml.c patch

### DISCUSSION THREAD

**Discussion by bmtwl** [xai-org/grok-1#29] [General]
Is the X AI team going to create a PR to add support for the Grok model to llama.cpp/ggml.c, or is that going to be a community effort?

**Comment by metatron1973** [comment]
Yea this is really cool ![image](https://github.com/xai-org/grok-1/assets/80199202/da2c92b2-b893-4d43-89de-79856a4333a8)

**Comment by trholding** [comment]
See: https://github.com/ggerganov/llama.cpp/pull/6204
